### PR TITLE
(node/lsstcam-fb01.cp.lsst.org) deploy host as "el7"

### PR DESCRIPTION
**Pull request by dtapiacl**

### CODE REVIEW

**Comment by dtapiacl** [comment]
This host behaves the same way as lsstcam-fcs as it uses canbus and will have to be deployed as EL7.

**Comment by glennmorris** [comment]
Added a couple of commits to enable the ccs canbus module.
The Puppetfile bump will lead to a trivial merge conflict at some point, sorry.
It requires a private hiera change too.

**Review by glennmorris — APPROVED** [review]

File: hieradata/node/lsstcam-fb01.cp.lsst.org.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+network::interfaces_hash:
+  enp3s0:  # fqdn
+    bootproto: "none"
+    ipaddress: "139.229.175.89"
```
Comment (jhoblitt) [inline hieradata/node/lsstcam-fb01.cp.lsst.org.yaml:5]: This address is missing from ipam.

**Review by jhoblitt — CHANGES_REQUESTED** [review]
This device is missing from ipam.

**Review by jhoblitt — APPROVED** [review]

**Pull request merged**